src/user-rank/user-rank.job.service.ts

```diff
@@ -0,0 +1,26 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { Injectable } from '@nestjs/common';
+import { GraphileWorkerPattern } from '../graphile-worker/enums/graphile-worker-pattern';
+import { GraphileWorkerService } from '../graphile-worker/graphile-worker.service';
+import { UserRankService } from './user-rank.service';
+@Injectable()
+export class UserRankJobService {
```

Comment:
Can we rename the file too? We use Jobs for graphile handers, service for primitive interactions with model tables, and loaders for classes that interact across different components.

```suggestion
export class UserRanksLoader {
```